### PR TITLE
LF-3079 Replace Digital Ocean Spaces with MinIO for the development environment

### PR DESCRIPTION
**Pull request by kathyavini**
**Description**

On beta & production, document upload and certification documents export both use a paid Digital Ocean Spaces service for storage. Developers currently can’t test these services locally without significant refactors to the user flow or using their own DIgital Oceans Spaces / AWS S3 accounts.

MinIO is a free and open-source drop-in replacement for AWS S3 that can be set up and run locally completely free of charge. Adding MinIO compatibility to the developer environment allows developers to locally test certification export and document upload and download in their local environments.

Instructions for using MinIO with the export server can be found on Confluence here: [Running the export server locally](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1190101039/The+export+jobs+pseudo-package#Running-the-export-server-locally)

These code changes were made specifically for use with the guide above, and remove hardcoded references to the Digital Ocean Spaces bucket in the 1) certification document export, 2) download of certification documents, and 3) document upload & download.

**A NOTE ABOUT IMAGES:** Image upload and download are handled by the LiteFarm Imaginary which I believe is a Digital Ocean Droplet and not a Digital Ocean Space. Image upload and download don't currently work locally, and this PR will not fix that. That feature would be a great addition for a developer to pick up in the future!

Jira link: https://lite-farm.atlassian.net/jira/software/c/projects/LF/issues/LF-3079

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update -- new variables have been documented in `.env.default`

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Other (please explain)

This branch has been used by myself, @SayakaOno and @Duncan-Brain to test the SurveyStack ticket [LF-2990](https://lite-farm.atlassian.net/browse/LF-2990).

**Checklist:**
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

[LF-2990]: https://lite-farm.atlassian.net/browse/LF-2990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Duncan-Brain** [comment]
@kathyavini 

Thanks for this one Joyce and @arcijsg for suggesting MinIO.

Definitely already used this a few times For surveystack and document export reviews and has worked well!

1. If there is any way to condense the Confluence document into a small ReadMe section, it could be good.

2. You mentioned it doesn't handle the images: could you write up a Jira ticket for that one to incorporate MinIO as well?

File: packages/webapp/src/containers/MediaWithAuthentication/index.jsx [hunk 1]
```diff
@@ -21,11 +21,15 @@ export function MediaWithAuthentication({
...
-        const url = new URL(fileUrl);
-        url.hostname = 'images.litefarm.workers.dev';
-        const response = await fetch(url.toString(), config);
-        const blobFile = await response.blob();
-        subscribed && setMediaUrl(URL.createObjectURL(blobFile));
+        if (import.meta.env.VITE_ENV === 'development') {
+          subscribed && setMediaUrl(fileUrl);
+        } else {
+          url.hostname = 'images.litefarm.workers.dev';
+          const url = new URL(fileUrl);
```
Comment (SayakaOno) [inline packages/webapp/src/containers/MediaWithAuthentication/index.jsx:28]: These two lines should be swapped, right...?

**Review by SayakaOno — COMMENTED** [review]
Thank you for working on this Joyce!
I left a comment :)

File: packages/webapp/src/containers/MediaWithAuthentication/index.jsx (hunk 1 above)
Comment (kathyavini) [reply packages/webapp/src/containers/MediaWithAuthentication/index.jsx:28]: Oh goodness 🤦‍♀️ Thank you

**Comment by SayakaOno** [comment]
In case you haven't seen it, you have a conflict in `packages/api/.env.default` 😉

**Review by Duncan-Brain — APPROVED** [review]
@kathyavini 

Looks great!

**Review by SayakaOno — APPROVED** [review]

**Pull request merged**